observability/opentelemetry/src/main/java/org/wildfly/extension/opentelemetry/OpenTelemetrySubsystemDefinition.java

```diff
@@ -163,6 +170,20 @@ protected OpenTelemetrySubsystemDefinition() {
         );
     }
 
+    public static String getExporter(OperationContext context, ModelNode model) throws OperationFailedException {
```

Comment:
You need to deal with the write attribute handler as well. I'll send an example in a few minutes.